### PR TITLE
Fix some small things in step-44's introduction.

### PR DESCRIPTION
**Pull request by drwells**

### CODE REVIEW

File: examples/step-44/doc/intro.dox [hunk 1]
```diff
@@ -109,29 +109,29 @@ such that
 		\qquad \text{and} \qquad
 	\dfrac{1}{2}[\mathbf{A} - \mathbf{A}^T] = \mathcal{W}:\mathbf{A} \, .
 @f]
-The fourth-order <code>SymmetricTensor</code> returned by identity_tensor() is $\mathcal{S}$.
+The fourth-order <code>SymmetricTensor</code> returned by identity_tensor() is $\mathcal{I}$.
```
Comment (jppelteret) [inline examples/step-44/doc/intro.dox:112]: This change is wrong. The original was correct.
> \dfrac{1}{2}[\mathbf{A} + \mathbf{A}^T] = \mathcal{S}:\mathbf{A}

File: examples/step-44/doc/intro.dox [hunk 2]
```diff
@@ -109,29 +109,29 @@ such that
...
+%Particles in the reference configuration are identified by the position vector $\mathbf{X}$.
 The configuration of the body at a later time $t>0$ is termed the current configuration, denoted $\Omega$, with particles identified by the vector $\mathbf{x}$.
 The nonlinear map between the reference and current configurations, denoted $\boldsymbol{\varphi}$, acts as follows:
 @f[
 	\mathbf{x} = \boldsymbol{\varphi}(\mathbf{X},t) \, .
 @f]
 The material description of the displacement of a particle is defined by
 @f[
-	\mathbf{U}(\mathbf{X},t) = \mathbf{x}(\mathbf{X},t) - \mathbf{X} \, .
+	\mathbf{U}(\mathbf{X},t) = \boldsymbol{\varphi}(\mathbf{X},t) - \mathbf{X} \, .
```
Comment (jppelteret) [inline examples/step-44/doc/intro.dox:127]: Although not incorrect, it is common in solid mechanics textbooks to just write "x = X + u". That is reflected here, so please change it back.

File: examples/step-44/doc/intro.dox [hunk 3]
```diff
@@ -109,29 +109,29 @@ such that
...
 @f]
 
 The deformation gradient $\mathbf{F}$ is defined as the material gradient of the motion:
 @f[
 	\mathbf{F}(\mathbf{X},t)
 		\dealcoloneq \dfrac{\partial \boldsymbol{\varphi}(\mathbf{X},t)}{\partial \mathbf{X}}
-		= \textrm{Grad}\ \mathbf{x}(\mathbf{X},t)
+		= \textrm{Grad}\ \mathbf{\varphi}(\mathbf{X},t)
```
Comment (jppelteret) [inline examples/step-44/doc/intro.dox:134]: This now reads exactly the same as the previous line?

File: examples/step-44/doc/intro.dox (hunk 2 above)
Comment (drwells) [reply examples/step-44/doc/intro.dox:127]: Sure.

File: examples/step-44/doc/intro.dox (hunk 3 above)
Comment (drwells) [reply examples/step-44/doc/intro.dox:134]: I thought the point of this sentence was to introduce `Grad`, but given your other point on notation I will change it back.

Comment (jppelteret) [reply examples/step-44/doc/intro.dox:134]: I can only presume that I meant to introduce `Grad` and work back towards the link between current position and displacement.

File: examples/step-44/doc/intro.dox (hunk 2 above)
Comment (jppelteret) [reply examples/step-44/doc/intro.dox:127]: Thanks!

File: examples/step-44/doc/intro.dox [hunk 4]
```diff
@@ -276,7 +276,7 @@ The fictitious Kirchhoff stress tensor $\overline{\boldsymbol{\tau}}$ is defined
 pressure in solid mechanics as
 $p = - 1/3 \textrm{tr} \boldsymbol{\sigma} = - 1/3 J^{-1} \textrm{tr} \boldsymbol{\tau}$.
 Here $p$ is the hydrostatic pressure.
-We make use of the pressure response throughout this tut (although we refer to it as the pressure).
+We make use of the pressure response throughout this tutorial (although we refer to it as the pressure).
```
Comment (jppelteret) [inline examples/step-44/doc/intro.dox:279]: Thanks for fixing this!

**Comment by drwells** [comment]
I now agree in retrospect that we should leave things as `x` which makes this patch quite small.

File: examples/step-44/doc/intro.dox (hunk 3 above)
Comment (jppelteret) [reply examples/step-44/doc/intro.dox]: Thank you for giving these points some thought :-)

**Comment by jppelteret** [comment]
@drwells Please feel free to merge this when the mandatory CI tests are complete. Since its just a docfix, we don't need to run in through the other testers.

**Pull request merged**